druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -3058,15 +3058,15 @@ private void testCountWithApproxDistinct(boolean approx, String sql, String expe
         + "(count(distinct \"user_id\") * 2) from \"wiki\"";
     wikiApprox(sql)
         .queryContains(druidChecker(druid))
-        .returnsUnordered("EXPR$0=-10590");
+        .returnsUnordered("EXPR$0=100");
 
     // Change COUNT(DISTINCT ...) to APPROX_COUNT_DISTINCT(...) and get
     // same result even if approximation is off by default.
     final String sql2 = "select (approx_count_distinct(\"user_id\") + 100) - "
         + "(approx_count_distinct(\"user_id\") * 2) from \"wiki\"";
     sql(sql2, WIKI)
         .queryContains(druidChecker(druid))
-        .returnsUnordered("EXPR$0=-10590");
+        .returnsUnordered("EXPR$0=100");
```

Comment:
Is this change of result expected (same as above)?

Reply:
fixed.